docs/source/conf.py

```diff
@@ -42,16 +42,16 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.githubpages",
     "sphinx_autodoc_typehints",
-    "sphinx_gallery.gen_gallery",
+    # "sphinx_gallery.gen_gallery",
```

Comment:
remove entirely? I still like the idea of examples in the documentation, but maybe there is a better way.

Reply:
I completely agree. At the same time I'm trying to really reduce the number of things we have to maintain. I'm also happy to leave it as-is. But as it stands, we only have one example which is a bit of a contrast to the tasks and notebooks, which pretty much have the same function.

Reply:
Sure we should't keep stale content, I think it was the first thing I wrote to explain the library and it is kind of nice to have tutorial style content in the docs, but we should maybe stick to well maintained notebooks for now. One advantage of these examples is that they automatically get build and executed as part of the build process so it is easy to detect failures and they can be exported / opened as Jupyter notebooks.

Reply:
In other words they should never be stale relative to the actual state of the library.